.github/workflows/publish-ruby.yaml

```diff
@@ -1,8 +1,5 @@
 name: Build Ruby
 on:
```

Comment:
The build now tags the binaries produces with both their platform (via the file extension) and their arch (new, in the filename)